provisioning/roles/php/tasks/main.yml

```diff
@@ -1,15 +1,15 @@
-- name: error when trying to use something else than PHP 5.6 on jessie
+- name: error when trying to use something else than PHP 5.6 or PHP 7.0 on jessie
```

Comment:
Concerning this whole file, I find it totally unreadable like that :( all those condition are difficult to understand and ansible display a line for each tasks even the ignored one so it doesn't make the output easy to understand either.

There is some condition inclusion in https://github.com/team-rawbot/rawbot-virtualization/blob/master/provisioning/roles/supervisor/tasks/main.yml maybe we can use the same trick here with two files :

* php5.yml
* php7.yml

Or even create a third file "wheezy.yml" to clean up further.

What do you think ?